update.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+git show "master:nsxiv.1" | groff -mandoc -Thtml > index.html
+awk '/<style type="text\/css">/,/<\/style>/ { if ( $0 ~ /<\/style>/ ) print "<link rel=\"stylesheet\" href=\"index.css\">"; next } 1' \
```

Comment:
I don't think we need to delete the existing style, just appending our style at the end should overwrite the default.